src/Card.js

```diff
@@ -26,26 +26,27 @@ class Card extends React.Component {
 	}
 
 	render() {
-		const {_id, imageSrc, title, description} = this.props;
+		const {_id, imageSrc, title, description, cards} = this.props;
```

Comment:
use 4 spaces or one tab here (not 8 spaces)

Reply:
I checked, everything is fine in the code. Possibly a github bug